fury/tests/test_actors.py

```diff
@@ -1492,3 +1492,12 @@ def test_marker_actor(interactive=False):
     report = window.analyze_snapshot(arr, colors=colors)
     npt.assert_equal(report.objects, 12)
 
+
+def test_actors_primitives_count():
+    centers = np.array([[1, 1, 1], [2, 2, 2]])
+    box_actor = actor.box(centers)
+    npt.assert_equal(get_primitives_count_from_actor(box_actor), len(centers))
+    box_actor = actor.sphere(centers, (1, 0, 0))
+    npt.assert_equal(get_primitives_count_from_actor(box_actor), len(centers))
+    box_actor = actor.sphere(centers, (1, 0, 0), use_primitive=True)
```

Comment:
Same as above
or above name with `prim_` prefix to differentiate the actors

Reply:
Thanks, @ganimtron-10 for your review! Helpful as always.